### PR TITLE
Support mindmap

### PR DESCRIPTION
**Pull request by a60814billy**
This PR add markmap-lib supported.
The example syntax:

<pre>

```markmap
# fruits

- banana
- mango
- melon
- orange
```
</pre>

螢幕快照 2020-04-11 上午5 32 36

related issue: #1407

### CODE REVIEW

File: app.js [hunk 1]
```diff
@@ -272,12 +272,18 @@ models.sequelize.sync().then(function () {
   } else {
     throw new Error('server still not ready after db synced')
   }
+}).catch(err => {
+  logger.error('Can\'t sync database');
```
Comment (jackycute) [inline app.js:276]:
```suggestion
  logger.error('Can\'t sync database')
```

Travis CI reported here has a extra semicolon.
https://travis-ci.com/github/hackmdio/codimd/jobs/318483790

Comment (a60814billy) [reply app.js]: fixed

**Comment by gtrias** [comment]
Is there any ETA for this feature to be shipped?

**Comment by Yukaii** [comment]
Thank you @a60814billy

**Comment by Yukaii** [comment]
@gtrias This will be included in the next release. We plan to have a new release in this two weeks.

**Pull request merged**